[ML] EUI Visual Refresh (tokens)

walterra opened the pull request:
## Summary

Part of #199715 (EUI Visual Refresh).

Recommend to review with white-space diff disabled: https://github.com/elastic/kibana/pull/203518/files?w=1

- All references to renamed tokens have been updated to use the new token name.
- All usage of color palette tokens and functions now pull from the theme, and correctly update to use new colors when the theme changes from Borealis to Amsterdam and vice versa.
- Migrated some data visualizer related SCSS to emotion, part of #140695.
- It makes use of EUI's own `useEuiTheme()` instead of our own hook variants. So this gets rid of `useCurrentEuiThemeVars()`, `useFieldStatsFlyoutThemeVars()`, `useCurrentEuiTheme()`, `useCurrentThemeVars()`.
- Renamed components used to edit Anomaly Detection jobs from `JobDetails`, `Detectors`, `Datafeed` to `EditJobDetailsTab`, `EditDetectorsTab`, `EditDatafeedTab` to make them less ambiguous.
- Added unit tests for `ner_output.tsx`.
- Adds checks to pick `euiColorVis*` colors suitable for the theme.

----

Some of our code used colors like `euiColorVis5_behindText`. In Borealis `*_behindText` is no longer available since the original colors like `euiColorVis5` have been updated to be suitable to be used behind text. For that reason and for simplicity's sake I removed the border from the custom badges we use to render NER items:

NER labels Amsterdam:

![CleanShot 2024-12-18 at 11 37 45@2x](https://github.com/user-attachments/assets/d82bca3a-2ad1-411a-94cd-748de6b4b0e9)

NER labels Borealis:

![CleanShot 2024-12-18 at 11 38 45@2x](https://github.com/user-attachments/assets/36987779-fab2-4ad7-8e31-6853d48079a1)


### Checklist

- [ ] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [x] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/259319)
* Commit: 6f414b4b34022f53c3161fbf93a6b834af575956

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/259319#0193b5ff-9104-49c2-a90b-7891dc8be935)

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259319","number":259319,"commit":"6f414b4b34022f53c3161fbf93a6b834af575956"}],"number":259319}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/259333)
* Commit: 14c4402fb122c9af2129bd630f7e38b560326e2a

### Failed CI Steps
* [FTR Configs #23](https://buildkite.com/elastic/kibana-pull-request/builds/259333#0193b62e-2254-4013-8614-8f2bf345f9de)
* [FTR Configs #23](https://buildkite.com/elastic/kibana-pull-request/builds/259333#0193b63f-1741-4f31-8891-7a339801d7f3)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/259333#0193b62e-2254-4013-8614-8f2bf345f9de) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/259333/jobs/0193b62e-2254-4013-8614-8f2bf345f9de/artifacts/0193b63e-fe30-467d-bc80-174850d4b226) FTR Configs #<span></span>23 / machine learning - data frame analytics outlier detection creation iowa house prices navigates through the wizard and sets all needed fields
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/259333#0193b63f-1741-4f31-8891-7a339801d7f3) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/259333/jobs/0193b63f-1741-4f31-8891-7a339801d7f3/artifacts/0193b646-8d01-4b91-ba48-6a4c19d9b260) FTR Configs #<span></span>23 / machine learning - data frame analytics outlier detection creation iowa house prices navigates through the wizard and sets all needed fields

### History
* :broken_heart: [Build #259319](https://buildkite.com/elastic/kibana-pull-request/builds/259319) failed 6f414b4b34022f53c3161fbf93a6b834af575956

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259333","number":259333,"commit":"14c4402fb122c9af2129bd630f7e38b560326e2a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259319","number":259319,"commit":"6f414b4b34022f53c3161fbf93a6b834af575956"}],"number":259333}
buildkite-pr-comment-->

elasticmachine commented:
Pinging @elastic/ml-ui (:ml)

mgadewoll commented on x-pack/platform/plugins/shared/ml/public/application/explorer/annotation_timeline.tsx:
```diff
@@ -79,7 +83,7 @@ export const AnnotationTimeline = <T extends { timestamp: number; end_timestamp?
         .attr('height', ANNOTATION_CONTAINER_HEIGHT)
         .attr('width', endingXPos - startingXPos)
-        .style('stroke', euiTheme.euiBorderColor)
+        .style('stroke', euiTheme.colors.borderBasePlain)
```
Question: `euiBorderColor` would map to `euiTheme.border.color` which is `borderBaseSubdued` in Borealis.
Should this border be stronger here or can it be the regular `euiTheme.border.color`? (same for following usages)

walterra replied:
👍 I'm using `euiTheme.border.color` now to match the original one ([904b773](https://github.com/elastic/kibana/pull/203518/commits/904b7739669a5e59cf861cd64c11dcfdef20ae33)).

peteharverson reviewed (approved):
Tested and visually LGTM - tested Borealis (dark theme) and Amsterdam (light theme).

andreadelrio commented on x-pack/platform/plugins/shared/ml/public/application/model_management/test_models/models/ner/ner_output.tsx:
```diff
@@ -11,53 +11,62 @@ import React from 'react';
     icon: 'visMapCoordinate',
-    color: 'euiColorVis1_behindText',
-    borderColor: 'euiColorVis1',
+    color: 'euiColorVis1',
```
Hi @walterra, visColors mapping has changed in Borealis (as seen [here](https://docs.google.com/document/d/1IAKbasq1nDfqd2IU3KdP8cwD3uCCAwkIekKRq7zgyWg/edit?usp=sharing)):

<img width="617" alt="image" src="https://github.com/user-attachments/assets/2f7bb7d9-12f9-4abe-aa68-df275e8388d9" />

So here if you want to preserve the blue you'll probably want to do  `euiColorVis2` instead.

mgadewoll replied:
@walterra @andreadelrio Fyi, changing colors like this means the current color in Amsterdam changes as well. (in Amsterdam it will be red now)
If that's not wanted, we'd need to use conditional color definitions.

andreadelrio commented on x-pack/platform/plugins/shared/ml/public/application/model_management/test_models/models/ner/ner_output.tsx:
```diff
@@ -11,53 +11,62 @@ import React from 'react';
     icon: 'questionInCircle',
-    color: 'euiColorVis7_behindText',
-    borderColor: 'euiColorVis7',
+    color: 'euiColorVis7',
```
```suggestion
    color: 'euiColorVis8',
```

andreadelrio commented on x-pack/platform/plugins/shared/ml/public/application/model_management/test_models/models/ner/ner_output.tsx:
```diff
@@ -11,53 +11,62 @@ import React from 'react';
   icon: 'questionInCircle',
-  color: 'euiColorVis5_behindText',
-  borderColor: 'euiColorVis5',
+  color: 'euiColorVis5',
```
```suggestion
  color: 'euiColorVis9',
```

andreadelrio commented on x-pack/platform/plugins/shared/ml/public/application/model_management/test_models/models/question_answering/question_answering_output.tsx:
```diff
@@ -59,15 +58,15 @@ function insertHighlighting(result: FormattedQuestionAnsweringResult, inputText:
   return (
     <EuiBadge
-      color={euiTheme.euiColorVis5_behindText}
+      color={euiTheme.colors.vis.euiColorVis5}
```
```suggestion
      color={euiTheme.colors.vis.euiColorVis9}
```

andreadelrio reviewed (changes requested):
Left some comments about visColors.

walterra commented:
@andreadelrio Thanks, that doc with the updated vis colors is quite helpful! I updated the colors in [ec09453](https://github.com/elastic/kibana/pull/203518/commits/ec094533421f8e0ade36a8c021ce8de5e20f027e) and [d395b77](https://github.com/elastic/kibana/pull/203518/commits/d395b7719fc77720caa7c90d904fbdc92d3dd4f8). Also updated the screenshot in the PR description.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/260458)
* Commit: d395b7719fc77720caa7c90d904fbdc92d3dd4f8

### Failed CI Steps
* [FTR Configs #27](https://buildkite.com/elastic/kibana-pull-request/builds/260458#0193c15a-6837-4733-8c6b-d6a752092bcb)
* [FTR Configs #27](https://buildkite.com/elastic/kibana-pull-request/builds/260458#0193c168-771a-4292-b09a-83d6372c1038)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260458#0193c15a-6837-4733-8c6b-d6a752092bcb) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260458/jobs/0193c15a-6837-4733-8c6b-d6a752092bcb/artifacts/0193c168-6490-4169-a3a5-6654bcdf2dd9) FTR Configs #<span></span>27 / machine learning - data frame analytics outlier detection creation iowa house prices navigates through the wizard and sets all needed fields
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260458#0193c168-771a-4292-b09a-83d6372c1038) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260458/jobs/0193c168-771a-4292-b09a-83d6372c1038/artifacts/0193c176-b5bf-4829-b840-6ac7c7428561) FTR Configs #<span></span>27 / machine learning - data frame analytics outlier detection creation iowa house prices navigates through the wizard and sets all needed fields

### History
* :broken_heart: [Build #259333](https://buildkite.com/elastic/kibana-pull-request/builds/259333) failed 14c4402fb122c9af2129bd630f7e38b560326e2a
* :broken_heart: [Build #259319](https://buildkite.com/elastic/kibana-pull-request/builds/259319) failed 6f414b4b34022f53c3161fbf93a6b834af575956

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260458","number":260458,"commit":"d395b7719fc77720caa7c90d904fbdc92d3dd4f8"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259333","number":259333,"commit":"14c4402fb122c9af2129bd630f7e38b560326e2a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259319","number":259319,"commit":"6f414b4b34022f53c3161fbf93a6b834af575956"}],"number":260458}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/260500)
* Commit: c31516bf8edb7024a8e864521e3efdba3fb2f2ee

### Failed CI Steps
* [FTR Configs #5](https://buildkite.com/elastic/kibana-pull-request/builds/260500#0193c1a2-50e1-484a-b1d2-6dcc80087d23)
* [FTR Configs #5](https://buildkite.com/elastic/kibana-pull-request/builds/260500#0193c1b4-40fb-40a1-ab58-1aa63a8d0b13)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260500#0193c1a2-50e1-484a-b1d2-6dcc80087d23) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260500/jobs/0193c1a2-50e1-484a-b1d2-6dcc80087d23/artifacts/0193c1b4-2308-4456-9cd4-f7589e627b70) FTR Configs #<span></span>5 / machine learning - data frame analytics regression creation electrical grid stability navigates through the wizard and sets all needed fields
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260500#0193c1b4-40fb-40a1-ab58-1aa63a8d0b13) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260500/jobs/0193c1b4-40fb-40a1-ab58-1aa63a8d0b13/artifacts/0193c1bc-6ab7-4274-aec4-be7ab4e6caec) FTR Configs #<span></span>5 / machine learning - data frame analytics regression creation electrical grid stability navigates through the wizard and sets all needed fields

### History
* :broken_heart: [Build #260458](https://buildkite.com/elastic/kibana-pull-request/builds/260458) failed d395b7719fc77720caa7c90d904fbdc92d3dd4f8
* :broken_heart: [Build #259333](https://buildkite.com/elastic/kibana-pull-request/builds/259333) failed 14c4402fb122c9af2129bd630f7e38b560326e2a
* :broken_heart: [Build #259319](https://buildkite.com/elastic/kibana-pull-request/builds/259319) failed 6f414b4b34022f53c3161fbf93a6b834af575956

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260500","number":260500,"commit":"c31516bf8edb7024a8e864521e3efdba3fb2f2ee"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260458","number":260458,"commit":"d395b7719fc77720caa7c90d904fbdc92d3dd4f8"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259333","number":259333,"commit":"14c4402fb122c9af2129bd630f7e38b560326e2a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259319","number":259319,"commit":"6f414b4b34022f53c3161fbf93a6b834af575956"}],"number":260500}
buildkite-pr-comment-->

andreadelrio reviewed (approved):
Style changes LGTM. Thanks!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/260518)
* Commit: cc2f35e191fba3360c99858f78dedfd30fffa73b

### Failed CI Steps
* [Jest Tests #9](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c1e7-187b-4386-9d95-1fb24c4b0cb6)
* [Jest Tests #9](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c212-bc44-437f-a832-bf548005fb72)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c1e7-187b-4386-9d95-1fb24c4b0cb6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c1e7-187b-4386-9d95-1fb24c4b0cb6/artifacts/0193c212-6c8d-490a-8c21-3c639e4f5501) Jest Tests #<span></span>9 / NER output getClassColor returns the correct color for class LOC
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c212-bc44-437f-a832-bf548005fb72) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c212-bc44-437f-a832-bf548005fb72/artifacts/0193c219-eb2e-4ce7-975e-b17127ea3f1b) Jest Tests #<span></span>9 / NER output getClassColor returns the correct color for class LOC
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c1e7-187b-4386-9d95-1fb24c4b0cb6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c1e7-187b-4386-9d95-1fb24c4b0cb6/artifacts/0193c212-6c8d-4ac2-92cf-a3677e5befe7) Jest Tests #<span></span>9 / NER output getClassColor returns the correct color for class MISC
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c212-bc44-437f-a832-bf548005fb72) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c212-bc44-437f-a832-bf548005fb72/artifacts/0193c219-eb2f-4d31-84fc-2ee3ede9ca26) Jest Tests #<span></span>9 / NER output getClassColor returns the correct color for class MISC
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c1e7-187b-4386-9d95-1fb24c4b0cb6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c1e7-187b-4386-9d95-1fb24c4b0cb6/artifacts/0193c212-6c8d-4f93-892a-f136e2b4574a) Jest Tests #<span></span>9 / NER output getClassColor returns the correct color for class PER
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c212-bc44-437f-a832-bf548005fb72) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c212-bc44-437f-a832-bf548005fb72/artifacts/0193c219-eb2e-4083-8256-2637c5018eb9) Jest Tests #<span></span>9 / NER output getClassColor returns the correct color for class PER
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c1e7-187b-4386-9d95-1fb24c4b0cb6) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c1e7-187b-4386-9d95-1fb24c4b0cb6/artifacts/0193c212-6c8d-44c2-bcf9-6c2ff850e67b) Jest Tests #<span></span>9 / NER output getClassColor returns the default color for an unknown class
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/260518#0193c212-bc44-437f-a832-bf548005fb72) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/260518/jobs/0193c212-bc44-437f-a832-bf548005fb72/artifacts/0193c219-eb2d-4840-9b04-06ca32c874dd) Jest Tests #<span></span>9 / NER output getClassColor returns the default color for an unknown class

### History
* :broken_heart: [Build #260500](https://buildkite.com/elastic/kibana-pull-request/builds/260500) failed c31516bf8edb7024a8e864521e3efdba3fb2f2ee
* :broken_heart: [Build #260458](https://buildkite.com/elastic/kibana-pull-request/builds/260458) failed d395b7719fc77720caa7c90d904fbdc92d3dd4f8
* :broken_heart: [Build #259333](https://buildkite.com/elastic/kibana-pull-request/builds/259333) failed 14c4402fb122c9af2129bd630f7e38b560326e2a
* :broken_heart: [Build #259319](https://buildkite.com/elastic/kibana-pull-request/builds/259319) failed 6f414b4b34022f53c3161fbf93a6b834af575956

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260518","number":260518,"commit":"cc2f35e191fba3360c99858f78dedfd30fffa73b"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260500","number":260500,"commit":"c31516bf8edb7024a8e864521e3efdba3fb2f2ee"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260458","number":260458,"commit":"d395b7719fc77720caa7c90d904fbdc92d3dd4f8"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259333","number":259333,"commit":"14c4402fb122c9af2129bd630f7e38b560326e2a"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259319","number":259319,"commit":"6f414b4b34022f53c3161fbf93a6b834af575956"}],"number":260518}
buildkite-pr-comment-->

rbrtj reviewed (commented):
It seems that some icons in the Analytics Map are not displayed properly (same for light theme):
![image](https://github.com/user-attachments/assets/1f981328-0057-4c2f-8d1b-c90a207c4ea1)

In the Data Visualizer, the color of single field top values now differs from the chart bar colors in the Borealis theme. This might be intentional, but I noticed the colors are the same in the Amsterdam theme:
![image](https://github.com/user-attachments/assets/9025bd5d-e709-4d0e-908b-35c30fc8e9f6)

The text on dark badges is now white, whereas it was previously dark. Not sure if this is intentional:
![image](https://github.com/user-attachments/assets/f9d8d0d7-f2ea-42d0-9ef3-6d73da33bd77)

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/261151)
* Commit: f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc

### Failed CI Steps
* [FTR Configs #78](https://buildkite.com/elastic/kibana-pull-request/builds/261151#0193d19c-5b25-48a3-88c8-a9df17feedf2)
* [FTR Configs #78](https://buildkite.com/elastic/kibana-pull-request/builds/261151#0193d1c6-8293-4f47-9b47-d207df4d7da7)
* [FTR Configs #102](https://buildkite.com/elastic/kibana-pull-request/builds/261151#0193d19c-5b3d-41b6-a2a1-77150c897067)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261151#0193d19c-5b3d-41b6-a2a1-77150c897067) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261151/jobs/0193d19c-5b3d-41b6-a2a1-77150c897067/artifacts/0193d1c1-df29-4657-b02e-8e7b25bf8a4e) FTR Configs #<span></span>102 / Spaces app space solution tour solution tour does not show the solution tour after deleting spaces and leave only the default
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261151#0193d19c-5b25-48a3-88c8-a9df17feedf2) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261151/jobs/0193d19c-5b25-48a3-88c8-a9df17feedf2/artifacts/0193d1c6-54b3-4872-92cc-77173e3fa135) FTR Configs #<span></span>78 / Upgrade Assistant Elasticsearch deprecation logs GET /api/upgrade_assistant/deprecation_logging /count should filter out the deprecation from Elastic products
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261151#0193d1c6-8293-4f47-9b47-d207df4d7da7) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261151/jobs/0193d1c6-8293-4f47-9b47-d207df4d7da7/artifacts/0193d1cc-aeaf-434a-9a20-4048c4194686) FTR Configs #<span></span>78 / Upgrade Assistant Elasticsearch deprecation logs GET /api/upgrade_assistant/deprecation_logging /count should filter out the deprecation from Elastic products

### History
* :yellow_heart: [Build #260656](https://buildkite.com/elastic/kibana-pull-request/builds/260656) was flaky 47ab04cab0b2504968f746d86b4de8a9ba92ca53
* :broken_heart: [Build #260518](https://buildkite.com/elastic/kibana-pull-request/builds/260518) failed cc2f35e191fba3360c99858f78dedfd30fffa73b
* :broken_heart: [Build #260500](https://buildkite.com/elastic/kibana-pull-request/builds/260500) failed c31516bf8edb7024a8e864521e3efdba3fb2f2ee
* :broken_heart: [Build #260458](https://buildkite.com/elastic/kibana-pull-request/builds/260458) failed d395b7719fc77720caa7c90d904fbdc92d3dd4f8
* :broken_heart: [Build #259333](https://buildkite.com/elastic/kibana-pull-request/builds/259333) failed 14c4402fb122c9af2129bd630f7e38b560326e2a
* :broken_heart: [Build #259319](https://buildkite.com/elastic/kibana-pull-request/builds/259319) failed 6f414b4b34022f53c3161fbf93a6b834af575956

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261151","number":261151,"commit":"f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260656","number":260656,"commit":"47ab04cab0b2504968f746d86b4de8a9ba92ca53"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260518","number":260518,"commit":"cc2f35e191fba3360c99858f78dedfd30fffa73b"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260500","number":260500,"commit":"c31516bf8edb7024a8e864521e3efdba3fb2f2ee"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260458","number":260458,"commit":"d395b7719fc77720caa7c90d904fbdc92d3dd4f8"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/259333","number":259333,"commit":"14c4402fb122c9af2129bd630f7e38b560326e2a"}],"number":261151}
buildkite-pr-comment-->

elasticmachine commented:
## :yellow_heart: Build succeeded, but was flaky
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/261266)
* Commit: 199f9898345a32b8959156d4af13da486a015c53

### Failed CI Steps
* [FTR Configs #6](https://buildkite.com/elastic/kibana-pull-request/builds/261266#0193d41c-6405-4680-b96e-325bbab18122)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/261266#0193d41c-6405-4680-b96e-325bbab18122) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/261266/jobs/0193d41c-6405-4680-b96e-325bbab18122/artifacts/0193d443-1ce2-4452-8a7a-5bbc3f316d66) FTR Configs #<span></span>6 / Rules Management - Prebuilt Rules - Prebuilt Rules Management @<span></span>ess @<span></span>serverless @<span></span>skipInServerlessMKI install_prebuilt_rules_from_real_package should install prebuilt rules from the package storage

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/199f9898345a32b8959156d4af13da486a015c53) | diff |
| --- | --- | --- | --- |
| `dataVisualizer` | 736 | 734 | -2 |
| `ml` | 2138 | 2137 | -1 |
| `transform` | 489 | 487 | -2 |
| total |  |  | -5 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/199f9898345a32b8959156d4af13da486a015c53) | diff |
| --- | --- | --- | --- |
| `@kbn/ml-kibana-theme` | 3 | 1 | -2 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/199f9898345a32b8959156d4af13da486a015c53) | diff |
| --- | --- | --- | --- |
| `aiops` | 621.5KB | 620.7KB | -814.0B |
| `dataVisualizer` | 616.4KB | 616.0KB | -333.0B |
| `ml` | 4.7MB | 4.7MB | +1.3KB |
| `transform` | 475.9KB | 475.4KB | -419.0B |
| total |  |  | -185.0B |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/199f9898345a32b8959156d4af13da486a015c53) | diff |
| --- | --- | --- | --- |
| `dataVisualizer` | 26.3KB | 26.3KB | -54.0B |
| `ml` | 78.4KB | 77.8KB | -608.0B |
| `transform` | 18.6KB | 18.5KB | -54.0B |
| total |  |  | -716.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/199f9898345a32b8959156d4af13da486a015c53) | diff |
| --- | --- | --- | --- |
| `@kbn/ml-kibana-theme` | 5 | 2 | -3 |


</details>


### History
* :green_heart: [Build #261223](https://buildkite.com/elastic/kibana-pull-request/builds/261223) succeeded 368ebd03b161ab0526df33561aa6a9f6298abb0c
* :broken_heart: [Build #261151](https://buildkite.com/elastic/kibana-pull-request/builds/261151) failed f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc
* :yellow_heart: [Build #260656](https://buildkite.com/elastic/kibana-pull-request/builds/260656) was flaky 47ab04cab0b2504968f746d86b4de8a9ba92ca53
* :broken_heart: [Build #260518](https://buildkite.com/elastic/kibana-pull-request/builds/260518) failed cc2f35e191fba3360c99858f78dedfd30fffa73b
* :broken_heart: [Build #260500](https://buildkite.com/elastic/kibana-pull-request/builds/260500) failed c31516bf8edb7024a8e864521e3efdba3fb2f2ee
* :broken_heart: [Build #260458](https://buildkite.com/elastic/kibana-pull-request/builds/260458) failed d395b7719fc77720caa7c90d904fbdc92d3dd4f8

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261266","number":261266,"commit":"199f9898345a32b8959156d4af13da486a015c53"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261223","number":261223,"commit":"368ebd03b161ab0526df33561aa6a9f6298abb0c"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261151","number":261151,"commit":"f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260656","number":260656,"commit":"47ab04cab0b2504968f746d86b4de8a9ba92ca53"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260518","number":260518,"commit":"cc2f35e191fba3360c99858f78dedfd30fffa73b"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260500","number":260500,"commit":"c31516bf8edb7024a8e864521e3efdba3fb2f2ee"}],"number":261266}
buildkite-pr-comment-->

walterra commented:
@rbrtj Thanks for spotting these issues! Fixed the analytics map icons in 8250674. Fixed the histogram chart colors in [d4346ac](https://github.com/elastic/kibana/pull/203518/commits/d4346ac3fc6f6f190976856055a8b7dc22f422e8). Please have another look!

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/261454)
* Commit: 8250674886f6262b7a584294937e9c00bbde7a9d

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/8250674886f6262b7a584294937e9c00bbde7a9d) | diff |
| --- | --- | --- | --- |
| `dataVisualizer` | 736 | 725 | -11 |
| `ml` | 2138 | 2137 | -1 |
| `transform` | 489 | 487 | -2 |
| total |  |  | -14 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/8250674886f6262b7a584294937e9c00bbde7a9d) | diff |
| --- | --- | --- | --- |
| `@kbn/ml-kibana-theme` | 3 | 1 | -2 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/8250674886f6262b7a584294937e9c00bbde7a9d) | diff |
| --- | --- | --- | --- |
| `aiops` | 621.5KB | 620.7KB | -747.0B |
| `dataVisualizer` | 616.4KB | 614.6KB | -1.8KB |
| `ml` | 4.7MB | 4.7MB | +1.4KB |
| `transform` | 475.9KB | 475.4KB | -419.0B |
| total |  |  | -1.6KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/8250674886f6262b7a584294937e9c00bbde7a9d) | diff |
| --- | --- | --- | --- |
| `aiops` | 17.3KB | 17.5KB | +208.0B |
| `dataVisualizer` | 26.3KB | 26.3KB | -54.0B |
| `ml` | 78.4KB | 77.8KB | -608.0B |
| `transform` | 18.6KB | 18.5KB | -54.0B |
| total |  |  | -508.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/8250674886f6262b7a584294937e9c00bbde7a9d) | diff |
| --- | --- | --- | --- |
| `@kbn/ml-kibana-theme` | 5 | 2 | -3 |


</details>


### History
* :yellow_heart: [Build #261266](https://buildkite.com/elastic/kibana-pull-request/builds/261266) was flaky 199f9898345a32b8959156d4af13da486a015c53
* :green_heart: [Build #261223](https://buildkite.com/elastic/kibana-pull-request/builds/261223) succeeded 368ebd03b161ab0526df33561aa6a9f6298abb0c
* :broken_heart: [Build #261151](https://buildkite.com/elastic/kibana-pull-request/builds/261151) failed f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc
* :yellow_heart: [Build #260656](https://buildkite.com/elastic/kibana-pull-request/builds/260656) was flaky 47ab04cab0b2504968f746d86b4de8a9ba92ca53
* :broken_heart: [Build #260518](https://buildkite.com/elastic/kibana-pull-request/builds/260518) failed cc2f35e191fba3360c99858f78dedfd30fffa73b
* :broken_heart: [Build #260500](https://buildkite.com/elastic/kibana-pull-request/builds/260500) failed c31516bf8edb7024a8e864521e3efdba3fb2f2ee

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261454","number":261454,"commit":"8250674886f6262b7a584294937e9c00bbde7a9d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261266","number":261266,"commit":"199f9898345a32b8959156d4af13da486a015c53"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261223","number":261223,"commit":"368ebd03b161ab0526df33561aa6a9f6298abb0c"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261151","number":261151,"commit":"f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260656","number":260656,"commit":"47ab04cab0b2504968f746d86b4de8a9ba92ca53"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260518","number":260518,"commit":"cc2f35e191fba3360c99858f78dedfd30fffa73b"}],"number":261454}
buildkite-pr-comment-->

rbrtj reviewed (approved):
Latest changes LGTM 👍

mgadewoll commented on x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/job_map/components/legend.tsx:
```diff
@@ -43,10 +43,10 @@ export const JobMapLegend: FC<{ hasMissingJobNode: boolean }> = ({ hasMissingJob
   const euiColorGhost = euiTheme.colors.ghost;
   const euiColorWarning = euiTheme.colors.warning;
-  const euiBorderThin = euiTheme.border.thin;
+  const euiBorderThin = euiTheme.border.width.thin;
```
Question: `euiBorderThin` does map to `euiTheme.border.thin` - Is this change expected/wanted because the usage cahnged for this being required to only be `1px` instead of `1px solid {euiTheme.border.color}`?

walterra replied:
Well spotted, you're right, this change was not intended, I reverted it in [dc5b902](https://github.com/elastic/kibana/pull/203518/commits/dc5b902e5874c0797aa38ab9d60976298c249302).

mgadewoll reviewed (approved):
Thanks for the additional changes and making sure Amsterdam colors have parity. 👏 
✅ Changes LGTM from EUI side

elasticmachine commented:
## :green_heart: Build Succeeded
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/261798)
* Commit: dc5b902e5874c0797aa38ab9d60976298c249302

### Metrics [[docs](https://www.elastic.co/guide/en/kibana/master/ci-metrics.html)]
#### Module Count
> Fewer modules leads to a faster build time


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/dc5b902e5874c0797aa38ab9d60976298c249302) | diff |
| --- | --- | --- | --- |
| `dataVisualizer` | 736 | 725 | -11 |
| `ml` | 2138 | 2137 | -1 |
| `transform` | 489 | 487 | -2 |
| total |  |  | -14 |

#### Public APIs missing comments
> Total count of every public API that lacks a comment. Target amount is 0. Run `node scripts/build_api_docs --plugin [yourplugin] --stats comments` for more detailed information.


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/dc5b902e5874c0797aa38ab9d60976298c249302) | diff |
| --- | --- | --- | --- |
| `@kbn/ml-kibana-theme` | 3 | 1 | -2 |

#### Async chunks
> Total size of all lazy-loaded chunks that will be downloaded as the user navigates the app


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/dc5b902e5874c0797aa38ab9d60976298c249302) | diff |
| --- | --- | --- | --- |
| `aiops` | 621.5KB | 620.7KB | -747.0B |
| `dataVisualizer` | 616.4KB | 614.6KB | -1.8KB |
| `ml` | 4.7MB | 4.7MB | +1.4KB |
| `transform` | 475.9KB | 475.4KB | -419.0B |
| total |  |  | -1.6KB |

#### Page load bundle
> Size of the bundles that are downloaded on every page load. Target size is below 100kb


| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/dc5b902e5874c0797aa38ab9d60976298c249302) | diff |
| --- | --- | --- | --- |
| `aiops` | 17.3KB | 17.5KB | +208.0B |
| `dataVisualizer` | 26.3KB | 26.3KB | -54.0B |
| `ml` | 78.4KB | 77.8KB | -608.0B |
| `transform` | 18.6KB | 18.5KB | -54.0B |
| total |  |  | -508.0B |

<details><summary>Unknown metric groups</summary>

#### API count
| id | [before](https://github.com/elastic/kibana/commit/dbf8818f88a5731b4614849e700a544ab8c06b95) | [after](https://github.com/elastic/kibana/commit/dc5b902e5874c0797aa38ab9d60976298c249302) | diff |
| --- | --- | --- | --- |
| `@kbn/ml-kibana-theme` | 5 | 2 | -3 |


</details>


### History
* :green_heart: [Build #261454](https://buildkite.com/elastic/kibana-pull-request/builds/261454) succeeded 8250674886f6262b7a584294937e9c00bbde7a9d
* :yellow_heart: [Build #261266](https://buildkite.com/elastic/kibana-pull-request/builds/261266) was flaky 199f9898345a32b8959156d4af13da486a015c53
* :green_heart: [Build #261223](https://buildkite.com/elastic/kibana-pull-request/builds/261223) succeeded 368ebd03b161ab0526df33561aa6a9f6298abb0c
* :broken_heart: [Build #261151](https://buildkite.com/elastic/kibana-pull-request/builds/261151) failed f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc
* :yellow_heart: [Build #260656](https://buildkite.com/elastic/kibana-pull-request/builds/260656) was flaky 47ab04cab0b2504968f746d86b4de8a9ba92ca53
* :broken_heart: [Build #260518](https://buildkite.com/elastic/kibana-pull-request/builds/260518) failed cc2f35e191fba3360c99858f78dedfd30fffa73b

cc @walterra

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261798","number":261798,"commit":"dc5b902e5874c0797aa38ab9d60976298c249302"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261454","number":261454,"commit":"8250674886f6262b7a584294937e9c00bbde7a9d"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261266","number":261266,"commit":"199f9898345a32b8959156d4af13da486a015c53"},{"buildStatus":{"state":"passed","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261223","number":261223,"commit":"368ebd03b161ab0526df33561aa6a9f6298abb0c"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/261151","number":261151,"commit":"f9f0a06bc2d88cdb39f3c515ebe38798bb202bdc"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/260656","number":260656,"commit":"47ab04cab0b2504968f746d86b4de8a9ba92ca53"}],"number":261798}
buildkite-pr-comment-->
